ice40/tests/blink/tinyfpga-bx.pcf

```diff
@@ -0,0 +1,10 @@
+set_io --warn-no-port LED3 J9
+set_io --warn-no-port LED4 E8
+set_io --warn-no-port LED5 J2
```

Comment:
Can you log a bug about adding support for the `--warn-no-port` parameter to the pcf converter?

Reply:
#202